tests/v2/conftest.py

```diff
@@ -1,124 +1,60 @@
 import os, pytest, uuid
 from tests.mock_requests import MockResponse
+from dotenv import load_dotenv
+from afs import models
 
-@pytest.fixture()
-def mocker_models(v2_env, mock_api_v2_resource):
-    from afs import models
-    yield models()
-
+load_dotenv()
 
 @pytest.fixture(scope='session')
-def test():
-    return uuid.uuid4()
-
-
-@pytest.fixture(scope='module')
-def model_name(test):
-    test_model = 'test_model.h5'
-    if os.path.exists(test_model):
-        os.remove(test_model)
-    with open(test_model, 'w') as f:
-        f.write(str(test))
+def model_file():
+  with open('unit_test_model', 'w') as f:
```

Comment:
Looks the indentation in this file is not correct. Please apply `black` or `yapf` to fix it.

Reply:
Although it is OK to run, I apply `black` to fix this bad looking.